examples/nginx-grafana-metabase/README.md

```diff
@@ -0,0 +1,9 @@
+# Visualizing Nginx Access Logs using Grafana and Metabase
+In my last [blog post](https://www.timeplus.com/post/analyzing-nginx-access-logs), I showed how Timeplus Proton can be used to analyze web traffic in production using just the log files generated by an Nginx web server. The article explored real-time analysis and historical analysis but the analysis wasn't visualized. 
+
+This article will remedy the lack of rich visualizations by showing how to do so using open source tools like [Grafana](https://grafana.com/) and [Metabase](https://www.metabase.com/).
```

Comment:
Can make this sentence more concise: 

This article remedies the lack of rich visualizations by showing how to use open-source tools like Grafana.

Reply:
Fixed.